src/index.html

```diff
@@ -1,12 +1,418 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="LogoMoyo"
+          class="header__logo"
+        >
+      </a>
+      <nav>
+        <ul class="header__nav">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#smartphones" class="header__nav-link">
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#laptops&computers"
+              class="header__nav-link collor_one"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#gadgets" class="header__nav-link">
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#tablets" class="header__nav-link">
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#photo" class="header__nav-link">
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#video" class="header__nav-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+            class="card__images"
+          >
+
+          <div class="card__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__code">
+            Product code: 195434
+          </div>
```

Comment:
Do not use div for text content - use semantic tags instead
h2-h5 for heading, p for other text
Fix all similar cases

Reply:
https://github.com/mate-academy/layout_catalog/pull/3145/files#r1196550136
not fixed